presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.intellij.lang.annotations.Language;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.accumulo.AccumuloQueryRunner.createAccumuloQueryRunner;
+import static com.facebook.presto.accumulo.AccumuloQueryRunner.dropTpchTables;
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestAccumuloDistributedQueries
+        extends AbstractTestDistributedQueries
+{
+    public TestAccumuloDistributedQueries()
+            throws Exception
+    {
+        super(createAccumuloQueryRunner(ImmutableMap.of(), true));
+    }
+
+    @AfterClass
+    public void cleanup()
+    {
+        dropTpchTables(queryRunner, getSession());
+    }
+
+    @Override
+    public void testAddColumn()
+            throws Exception
+    {
+        try {
+            // TODO Adding columns via SQL are not supported until adding columns with comments are supported
+            super.testAddColumn();
```

Comment:
Why you call `super.testAddColumn` test while you don't support `alter table add column`?